src/ChinookSnippets/Chinook.DynamicMvvm.Combined/DynamicCommandWithParameterFromAction.snippet

```diff
@@ -0,0 +1,45 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>DynamicCommand with Parameter from Action</Title>
```

Comment:
I think we should keep the existing name and add the additional suffix.
i.e. This should be `DynamicCommandFromActionWithParameter`.
Make sure you rename the one from task and both file.